src/cmd-build

```diff
@@ -118,24 +109,26 @@ ostreesetup --nogpg --osname=coreos --remote=coreos --url=file:///mnt/ostree-rep
 EOF
 
 imageprefix=${name}-${version}-${image_genver}
-/usr/lib/coreos-assembler/virt-install --dest=$(pwd)/${imageprefix}-base.qcow2 \
-               --create-disk --kickstart $(pwd)/local.ks --kickstart-out $(pwd)/flattened.ks \
+/usr/lib/coreos-assembler/virt-install --dest=$(pwd)/tmp/${imageprefix}-base.qcow2 \
+               --create-disk --kickstart $(pwd)/tmp/local.ks --kickstart-out $(pwd)/tmp/flattened.ks \
                --location ${workdir}/installer/*.iso --console-log-file $(pwd)/install.log \
```

Comment:
should this be `$(pwd)/tmp/install.log`

Reply:
I decided to keep it, but could be convinced to drop it too.